### ISSUE TITLE
[Bug Bash] First pressing of “down” button on keyboard in the “version” drop-down box of PM UI will bring null value

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#11007]
## Details about Problem
VS Version: Main\31507.353
NuGet Version: Dev\6.0.0.140
OS Version: Windows-10-Enterprise-21H1
## Repro steps
1.	Launch VS and create a solution [e.g. C# Console App (.NET Framework 4.8)].
2.	Open the solution-level/project-level PM UI.
3.	Go to “Browse” tab (or other tabs) and press tab through the PM UI.
4.	Press “down” button on keyboard when the focusing is on the “version” drop-down box.
## Expected
The first pressing of “down” button on keyboard should not bring null value.
## Actual
The first pressing of “down” button on keyboard will bring null value as below screenshot.
![image](https://user-images.githubusercontent.com/20632625/124901306-76153080-e014-11eb-996c-52cbee6b588e.png)
## Notes:
1.	The version will show correctly from the second pressing of “down” button.
2.	It seems a "not a regression" bug since it did repro on D16.8\30420.98 with implicit NuGet Client Dev\5.8.0.6776.

**Comment by donnie-msft** [comment]
Install not required. The separator is added as an item in the list, but for some reason it's only treated as an item the first time. Opening/closing the dropdown once causes the item to no longer appear when arrow keying.

**Comment by v-luzh** [comment]
@donnie-msft, yes, there is no need to install. Updated the bug.

**Comment by donnie-msft** [comment]
Our code appears to be correct, but WPF for some reason allows disabled items to be selected via keyboard until the dropdown is expanded once.

I've filed a WPF bug here: https://github.com/dotnet/wpf/issues/4929. The labels applied make me think this will not be fixed soon.

Rather than play with some hacks to mitigate this, and since no customers have expressed concern, I believe this should be closed as "not a bug". 
However, I did find that Project PM UI has an enabled Install button while this is selected. It doesn't actually work, but should be disabled. Filed that here https://github.com/NuGet/Home/issues/11096
